doc/user_manual/rom.tex

```diff
@@ -3295,3 +3571,642 @@ \subsubsection{DMD}
+<Simulation>
+  ...
+  <Models>
+    ...
+    <ROM name='aUserDefinedName' subType='whatever'>
+      <Features>X,Y</Features>
+      <Target>Z</Target>
+      <loss>mean_squared_error</loss>
+      <metrics>accuracy</metrics>
+      <batch_size>4</batch_size>
+      <epochs>4</epochs>
+      <num_classes>2</num_classes>
+      <validation_split>0.25</validation_split>
+      <optimizerSetting>
+        <optimizer>Adam</optimizer>
+        ...
+      </optimizerSetting>
+      <WhateverLayer1 name="layerName1">
+        ...
+      </WhateverLayer1>
+      ...
+      <WhateverLayerN name="layerNameN">
+        ...
+      </WhateverLayerN>
+      <layer_layout>layerName1, ..., layerNameN</layer_layout>
+    </ROM>
+    ...
+  </Models>
+  ...
+</Simulation>
+\end{lstlisting}
+
+As shown in above example, in addition to the common subnodes \xmlNode{Target} and \xmlNode{Features}, the \xmlNode{ROM} of DNNs
+can be initialized with the following children:
+\begin{itemize}
+  \item \xmlNode{loss}, \xmlDesc{string or comma separated string, optional field}, if the model has multiple outputs, you can use a different
+    loss on each output by passing a list of losses. The loss value that will be minimized by the model will then
+    be the sum of all individual losses. Available loss functions include \textit{mean\_squared\_error},
+    \textit{mean\_absolute\_error}, \textit{mean\_absolute\_percentage\_error}, \textit{mean\_squared\_logarithmic\_error},
+    \textit{squared\_hinge}, \textit{hinge}, \textit{categorical\_hinge}, \textit{logcosh}, \textit{categorical\_crossentropy},
+    \textit{sparse\_categorical\_crossentropy}, \textit{binary\_crossentropy}, \textit{kullback\_leibler\_divergence},
+    \textit{poisson}, \textit{cosine\_proximity}.
+  \default{mean\_squared\_error}
+  \item \xmlNode{metrics}, \xmlDesc{string or comma separated string, optional field}, list of metrics to be evaluated by
+    the model during training and testing. available metrics include
+    \textit{binary\_accuracy}, \textit{categorical\_accuracy}, \textit{sparse\_categorical\_accuracy},
+    \textit{top\_k\_categorical\_accuracy}, 
+    %
+    \textit{sparse\_top\_k\_categorical\_accuracy}.
+  \default{accuracy}
+  \item \xmlNode{batch\_size}, \xmlDesc{integer, optional field}, number of samples per gradient update.
+  \default{20}
+  \item \xmlNode{epochs}, \xmlDesc{integer, optional field}, number of epochs to train the model. An epoch
+    is an iteration over the entire training data.
+  \default{20}
+  \item \xmlNode{num\_classes}, \xmlDesc{positive integer, optional field}, dimensionality of the output space of given classifier.
+  \default{1}
+  \item \xmlNode{validation\_split}, \xmlDesc{float between 0 and 1, optional field}, fraction of the training data to
+    be used as validation data.
+  \default{0.25}
+  \item \xmlNode{plot\_model}, \xmlDesc{boolean, optional field}, if true the DNN model constructed by RAVEN will be
```

Comment:
I have add the output to the metadata except the model structure plot. I may need to move this to ROM print, or remove it, since this capability requires additional libraries ```pydot-ng``` and ```graphviz```. For now, I would like to keep it there.